File: python/kasirmini.py
nama = input("masukan nama barang: ")
harga = int(input("masukan harga barang :"))
jumlah =int(input("masuka jumlah beli :"))

total = harga*jumlah

if total >56500:
    diskon = total * 0,
else:
    diskon = 0

total_bayar=total - diskon

print("\n====== STRUK PEMBELIAN =====")
print(f"nama barang : {nama}")
print(f"harga satuan : {harga}")
print(f"jumlah beli : {jumlah}")
print(f"total harga : {total}")
print(f"diskon : {diskon}")
print(f"total bayar : {total_bayar}")
print("==============================")

File: php/kasirmini.php
<?php
$nama = readline("masukan nama barang: ");
$harga = readline("masukan harga barang: ");
$jumlah = readline("masukan jumlah beli: ");

$total = $harga*$jumlah;

if ($total>56500)
    echo $diskon = $total*0,5;
else
    echo $diskon =0;

$total_bayar = $total-$diskon;

echo"\n====== STRUK PEMBELIAN =====\n";
echo"nama barang : {$nama}";
echo"harga satuan : {$harga}";
echo"jumlah beli : {$jumlah}";
echo"total harga : {$total}";
echo"diskon : ${diskon}";
echo"total bayar : {$total_bayar}";
echo"==============================";
?>